data/types/blood/blood.go

```diff
@@ -51,3 +53,46 @@ func (b *Blood) LegacyIdentityFields() ([]string, error) {
 		types.GetLegacyTimeField(b.Time),
 	)
 }
+
+func (b *Blood) GetRawUnitsAndValue() (*string, *float64, error) {
```

Comment:
[nit] Consider reversing the order of units and value to match the other functions such as `NormalizeValueForUnits`.